Libraries/LibJS/Bytecode/Generator.cpp

```diff
@@ -882,6 +882,10 @@ void Generator::emit_set_variable(JS::Identifier const& identifier, ScopedOperan
 {
     if (identifier.is_local()) {
         auto local_index = identifier.local_index();
+        if (initialization_mode == Bytecode::Op::BindingInitializationMode::Set && identifier.declaration_kind() == DeclarationKind::Const) {
+            emit<Bytecode::Op::ThrowInvalidAssignToConst>();
```

Comment:
issue: Since this is an unconditional throw, perform_needed_unwind should be called.
suggestion:
```suggestion
            perform_needed_unwinds<Bytecode::Op::Throw>();
            emit<Bytecode::Op::ThrowInvalidAssignToConst>();
```